ocaml/typing/oprint.ml

```diff
@@ -975,16 +959,12 @@ and print_out_type_decl kwd ppf td =
     print_unboxed
 
 and print_simple_out_gf_type ppf (ty, gf) =
-  let locals_enabled = Language_extension.is_enabled Local in
   match gf with
   | Ogf_global ->
-    if locals_enabled then begin
+      (* Printing modalities regardless of `mode` extension *)
```

Comment:
I think it might be confusing if mode extension is disabled but modes are still printed - especially when the counterpart (parsing) is controlled by the extension. Basically I want to deter a future developer from attempting to make printing depending on extension.

Reply:
Good point - I will elaborate.